### ISSUE TITLE
Localization problems?

### ISSUE THREAD

**Issue by AdamJachocki** [PiranhaCMS/piranha.core#1538]
Hi, today I wanted to add Piranha to my EXISTING web page (MVC). After some struggling, Piranha Manager started to work. BUT! I "lost" my translations for my site. Piranha has to do something, which I would say is weird.

Until now I had to translate my views like this:

```
@using Microsoft.AspNetCore.Mvc.Localization
@using MyApp.LocalizedResources
@inject IHtmlLocalizer<WebLangRes> WebLocalizer

<p>@WebLocalizer[nameof(WebLangRes.Hello)]</p>
```

When I started my app after configuring Piranha, my pages lost its translations. After some tries I found out that now I just have to do:

```
<p>@WebLangRes.Hello</p>
```

Why is that? Is there something I could do to work this out? In the end I would be able to translate my razor views as I used to.

**Comment by tidyui** [comment]
Hi there! The manager performs the following calls on the `IMvcBuilder` when setting up the application:

~~~
.AddViewLocalization()
.AddDataAnnotationsLocalization()
~~~

What setup did you have in your `Startup` for localization?

**Comment by AdamJachocki** [comment]
So, any news?

**Comment by jensbrak** [comment]
I have just implemented localization for a little module I made but haven't found anything that would suggest the issue described here. However, there are certain aspects of localizing a manager compatible module that are far from obvious.

I tried to replicate the way the Manager use localization. From what I gather, the manager use this construct to get translations `@Localizer.Page["Your page title"]"` as opposed to `nameof()`. Then again - the manager use the `IStringLocalizer` from `Microsoft.Extensions.Localization` as opposed to what you @AdamJachocki use which is the `HtmlLocalizer` of `Microsoft.AspNetCore.Mvc.Localization`.

I haven't really digged into the relation between the two but if I am not totally wrong we do look at two different localizers here. Just a note on my behalf. I still have to get mine to work, manager goes localized but not my module - despite all efforts.
